prow/plugins/approve/approvers/README.md

```diff
@@ -129,13 +132,15 @@ K8s-bot updates comment:
 	You can indicate your approval by writing `/approve` in a comment
 	You can cancel your approval by writing `/approve cancel` in a comment
 
-**Case (b)**: *approver3* is NOT an approver of the PR. S/he writes `/approve`
+### Step 3:
```

Comment:
Since we changed this from case (b), to the step following what was previously case (a), we need to include the changes from the previous step. Namely, `approver1` approved the PR which satisfied the `/A/B/E/OWNERS` requirement. That should be reflected in the details section.

Reply:
Done consistently.